Frontend/ui-driver/src/Screens/ReferralScreen/View.purs

```diff
@@ -347,8 +348,19 @@ dateAndTime push state =
         , gravity RIGHT
         , weight 1.0
         ]
-    ]
+  ]
 
+leaderBoardRanksCover :: forall w . (Action -> Effect Unit) -> ST.ReferralScreenState -> PrestoDOM (Effect Unit) w
+leaderBoardRanksCover push state = 
+  swipeRefreshLayout
+  [ width MATCH_PARENT
+  , height MATCH_PARENT
+  , orientation VERTICAL
+  , onRefresh push $ const RefreshScreen
+  , id (getNewIDWithTag "referral_refresh_view")
```

Comment:
don't use Snake Case

Reply:
Used "ReferralRefreshView" instead of "referral_refresh_view"